classes/class-kp-logger.php

```diff
@@ -29,10 +29,7 @@ public static function log( $data ) {
 		$kp_settings = get_option( 'woocommerce_klarna_payments_settings', array() );
 		if ( 'no' !== $kp_settings['logging'] ) {
 			$message = self::format_data( $data );
-			if ( empty( self::$log ) ) {
-				self::$log = new WC_Logger();
-			}
-			self::$log->add( 'klarna_payments', wp_json_encode( $message ) );
+			KP_WC()->logger()->debug( wp_json_encode( $message ) );
```

Comment:
Question: This will log everything as debug rather then info which is the current level. Are we sure we want to change that for every request?